src/index.tsx

```diff
@@ -21,31 +22,48 @@ const Content: VFC<{ serverAPI: ServerAPI }> = ({ serverAPI }) => {
   const [ tailscaleToggle, setTailscaleToggle ] = useState(false);
 
   backend.resolvePromise(backend.getTailscaleState(), setTailscaleToggle);
-  
+
   const toggleTailscale = async(switchValue: boolean) => {
     setTailscaleToggle(switchValue);
     await serverAPI.callPluginMethod((switchValue) ? 'up' : 'down', {});
   }
 
+  const [tailscaleDevices, setTailscaleDevices] = useState<string | undefined>(undefined)
+  const updateDevices = ()=>{
+    backend.getTailscaleDevices().then((devices)=>{
+      setTailscaleDevices(devices.result)
+    })
+  }
+  useEffect(()=>{
+    if (tailscaleToggle){
+      updateDevices()
+    }
+  },[tailscaleToggle])
+  setInterval(()=>updateDevices(), 5000)
+
   return (
-    <PanelSection title="Settings">
+      <PanelSection title="Settings">
         <PanelSectionRow>
           <ToggleField
-          bottomSeparator='standard'
-          checked={tailscaleToggle}
-          label='Toggle Tailscale'
-          description='Toggles Tailscale On or Off'
-          onChange={toggleTailscale} />
+              bottomSeparator='standard'
+              checked={tailscaleToggle}
+              label='Toggle Tailscale'
+              description='Toggles Tailscale On or Off'
+              onChange={toggleTailscale} />
+        </PanelSectionRow>
+        <PanelSectionRow>
+          <div>Tailscale Devices</div>
+          {tailscaleDevices}
```

Comment:
Happy to take others' guidance here, didn't find much in https://github.com/SteamDeckHomebrew/decky-plugin-database/tree/main/plugins that showcases complex text layouts (eg. a table?) to better show this data